src/misc/appPackager.ts

```diff
@@ -12,8 +14,6 @@ export class AppPackager {
         silent: true,
         ignore: [
             '**/README.md',
-            '**/package-lock.json',
```

Comment:
Let's not include this file, as we don't look at the `package-lock.json` file and we should reduce the amount of usage we should do.